server/controllers/notificationController.js

```diff
@@ -51,33 +51,60 @@ const notificationController = {
       })
       .then((notify) => {
         if (!notify) {
-          return res.redirect('/api/v1/notifications');
+          return res.status(404).json({
+            status: 'error',
+            error: 'Notification does not exist'
+          });
         }
         notify.isRead = true;
         notify.save()
           .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the server'));
-        const route = {
-          article: `/api/v1/articles/${notify.notifiableId}`,
-          user: `/api/v1/users/${notify.notifiableId}`,
-        };
-        return res.redirect(route[notify.notifiable]);
+        return res.status(200).json({
+          status: 'success',
+          notify
+        });
       })
-      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
+      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the server'));
   },
   markAsRead: (req, res) => {
+    const { readStatus } = req.body;
+
     Notifications
-      .findById(req.params.notifyId)
+      .scope(null)
+      .findByPk(req.params.notifyId)
```

Comment:
Why do we have to use `.findByPk` over `. findById`

Reply:
I used `findByPk` because `findById` is deprecated.

Reply:
👍